### ISSUE TITLE
SVG's "null"

### ISSUE THREAD

**Issue by MarkMurphy** [FWeinb/grunt-svgstore#95]
I have an svg folder with one svg user.svg. The output gives me the following:

```xml
<svg xmlns="http://www.w3.org/2000/svg"><symbol id="icon-user"><title>user</title>null</symbol></svg>
```

My grunt task looks like this:

```js
svgstore: {
  options: {
    prefix : 'icon-'
  },
  default: {
    files: {
      "app/assets/images/icons.svg": [
        "app/assets/images/svg/*.svg"
      ]
    }
  }
}

**Comment by JnBrewsterGT** [comment]
I have this very problem,  Do you have the solution?  What am I doing wrong?

**Comment by MarkMurphy** [comment]
Find an svg that does work and then just copy your path data into it. I think it's something to do with the tags preceding the opening <svg> or the attributes on it

**Comment by MarkMurphy** [comment]
If there's still a problem after that then it's probably something in the path data

**Comment by JnBrewsterGT** [comment]
Yes, thank you.  That was it.  I opened up the svg file in an editor and saw the non ascii characters.  I used a process to remove it and it worked after that.

I was going to post what I had found.

Thanks for getting back to me.

**Comment by MarkMurphy** [comment]
Yeh, so I guess it would be helpful if this project did one of two things:

1.) Ideally, validate the SVG and inform the user of any problems.
2.) If something fails, display an error message to the user and direct them to a validator. The error message might provide a couple of examples of common issues.

Or maybe both, if there appears to be a problem with the input or output of the svg, validate it then to try inform the user why it's failed. If that fails or is uncertain, fallback to just displaying a generic error message and a link to a validator.

**Comment by JnBrewsterGT** [comment]
+1
